src/pages/Registration/Steps/Documentation/Form/useSubmit.ts

```diff
@@ -30,6 +32,7 @@ export default function useSubmit() {
     level,
     hqCountry,
     activeInCountries,
+    CashEligible,
```

Comment:
Shouldn't this field be submitted as well? Currently its value is just set and then forgotten.